spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/client/loadbalancer/impl/scoped/ScopedDiscoveryClient.java

```diff
@@ -0,0 +1,12 @@
+package org.springframework.cloud.client.loadbalancer.impl.scoped;
+
+import org.springframework.cloud.client.ServiceInstance;
+
+import java.util.List;
+
+/**
+ * @author Spencer Gibb
+ */
+public interface ScopedDiscoveryClient {
+	List<ServiceInstance> getInstances();
```

Comment:
should the caller anticipate runtime exceptions or are they protected from them?